ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -279,3 +275,22 @@ extension CommandBar: UIScrollViewDelegate {
         updateShadow()
     }
 }
+
+/// A UIView subclass that updates its mask frame during layoutSubviews. By default, the layer mask
+/// is not hooked into auto-layout and will not update its frame if its parent frame changes size. This implementation
+/// fixes that.
+private class MaskedView: UIView {
+    init() {
+        super.init(frame: .zero)
+    }
+
+    @available(*, unavailable)
+    required init(coder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
```

Comment:
Don't need any of this init stuff since we don't do any special behavior on init?

Reply:
Correct. Removed.